lupa/_lupa.pyx

```diff
@@ -1666,14 +1678,24 @@ cdef _LuaTable py_to_lua_table(LuaRuntime runtime, lua_State* L, items):
     check_lua_stack(L, 5)
     old_top = lua.lua_gettop(L)
     lua.lua_newtable(L)
+    cdef int lua_table_ref = lua.lua_gettop(L) # the index of the lua table which we are filling
     # FIXME: how to check for failure?
```

Comment:
I think this comment was referring to the line above. And it's probably outdated, because we can't check for failures, Lua would just jump to the error handler.
```suggestion
    # FIXME: handle allocation errors
    cdef int lua_table_ref = lua.lua_gettop(L) # the index of the lua table which we are filling
```